packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -207,34 +227,72 @@ const DesignPicker: React.FC< DesignPickerProps > = ( {
 } ) => {
 	const hasCategories = !! Object.keys( categorization?.categories || {} ).length;
 	const filteredDesigns = useFilteredDesigns( designs, categorization );
-
-	// Pick design
+	const features = [ 'blog', 'magazine', 'portfolio', 'podcast', 'store' ];
```

Comment:
There is not `newsletter` in the list of categories. I have used `magazine` instead.

Reply:
Looks like we have to return the newsletter from the endpoint. I remember we didn't want to show the newsletter category before but I forgot the reason...

Reply:
There are only 2 recommended newsletter designs so the `newsletter` category is hidden. We have to add more designs to display it.

Reply:
Ola is working on adding more themes to `newsletter`. It's fine if we only render 4 features in the meantime!